admin-dev/themes/new-theme/js/app/pages/stock/components/overview/movement-type.vue

```diff
@@ -57,6 +57,7 @@
     },
     methods: {
       sendQty(): void {
+        this.$store.state.hasQty = false;
```

Comment:
It looks pretty strange, is it the sure way of disabling the button? I'm worried of side effects of this, it has quantity but you want to disable the button

Reply:
The only thing using the hasQty is the logic enable and disable button. I had the same hunch as yourself and had a test 'hasQty' has a bool so it's not storing the quantity and the request is sent but the button is now turned off before hand. am on my phone at the moment I can put a gif up later if needed.

Reply:
What about renaming the state like `this.$store.state.isButtonEnabled` maybe it would be clearer for the future?

Overwise the screen is welcome anyway, it greatly increases the quality of the pull request and it makes the QA easier!

Thanks for the work anyway :)